odoo/addons/base/views/res_partner_views.xml

```diff
@@ -151,7 +151,7 @@
                 <sheet>
                     <div class="oe_button_box" name="button_box"/>
                     <widget name="web_ribbon" title="Archived" bg_color="bg-danger" attrs="{'invisible': [('active', '=', True)]}"/>
-                    <field name="image_1920" widget='image' class="oe_avatar" options='{"preview_image": "image_128"}'/>
+                    <field name="image_1920" widget='image' class="oe_avatar" placeholder="/base/static/img/avatar_grey.png" options='{"preview_image": "image_128"}'/>
```

Comment:
Shouldn't this be done at model level through `_get_placeholder_filename` override?

Reply:
same